### DISCUSSION TITLE
Unable to login admin panel

### DISCUSSION THREAD

**Discussion by fpscan** [globaleaks/globaleaks-whistleblowing-software#4005] [Q&A]
Hi Everyone,

I have a environment where I don't know when and who set up Globaleaks but it's been there since 2021. I have been asked to check the environment recently. I tried to login(internernaldomain.com/#/admin) panel with the passwords I usually use, but it didn't worked. Then I tried to login the virtual machine but unable to login that too. So I managed to crack the root and log in as root and tried this command; gl-admin resetpass admin but it didn't worked. So I was wondering if there anything I can do from here. I tried to check the 8443, 8082, 8083 and 9050 ports to check but couldn't get in.

![image](https://github.com/globaleaks/GlobaLeaks/assets/4651944/fb4013f5-84d2-41bd-af92-4e15ae63c44a)

So I was just hoping to reset admin password and login in webui. I don't have db, admin or any other informations. Unlucky, no one knows anything about it.

**Comment by evilaliv3** [comment]
Hello @fpscan , thank you for the question.

Actually i think your situation could not be recovered.
GlobaLeaks encrypts data securely and bruteforcing password is not an option as it is designed to make bruteforce unfeasible.
If the systems hosts reports you really need the password of users or their account recovery key or you wont be able to recover any data.
If instead you do not have any data i suggest you just install a new fresh system.

You may reach out to me on community.globaleaks.org if helpful.
